core-web/libs/dot-primeng-theme-styles/src/scss/_variables.scss

```diff
@@ -152,3 +152,6 @@ $form-width: 400px;
 
 // CONTENT TYPES
 $content-type-field-height: 74px;
+
+// BLOCK EDITOR BASE FONT SIZE
+$dot-editor-size: 16px;
```

Comment:
Can we have this in a file inside the block editor?

Reply:
Sure!